### PR TITLE
fix: updated tab titles to get title using react-helmet

### LINKED ISSUE #1041: Use React-Helmet to manage the document head
Instead of manually declaring things like `document.title`, we should probably switch to using `react-helmet` for this purpose. I have to wonder if it wouldn't also help with things like SEO?

### PR DESCRIPTION
**Pull request by dotslashbit**
## Description

fixes #1041 

now tab titles are getting titles using react-helmet

### CODE REVIEW

**Comment by dotslashbit** [comment]
@anth-volk waiting for #1128 to be merged as its updating that current law logic and also updating HouseHold tab titles using react-helmet. So after #1128 merge, I'll rebase this PR with master and then put this PR for review.

**Comment by anth-volk** [comment]
Will review tomorrow

**Review by anth-volk — APPROVED** [review]
Thanks for this @dotslashbit! Approved.

**Pull request merged**
